internal/command/command.go

```diff
@@ -42,12 +42,12 @@ func FlagKeyCost(key string) int {
 	return len(key) + strings.Count(key, ".")
 }
 
-func DelimCost(delims []rune) int {
+func DelimCost(delims []byte) int {
```

Comment:
When migrating to `cobra`, it made more sense to pass delimiters around as bytes instead of runes